app/posts/publish-teacher-training-courses/2019-10-22-accredited-bodies-research-round-2.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Accredited bodies research round 2 - reporting
+description: Continued research with Accredited bodies to determine reporting needs
+date: 2019-10-22
+related:
+  items:
+  - text: Lookback sessions
+    href: https://lookback.io/dfe-digital/find-accrediting-bodies-round-2
+  - text: Research playback
+    href: https://docs.google.com/presentation/d/1ChxmFgCA3MHBxtcuDwtkiUeNdYBXs2prmeqa9jtQuk8/edit#slide=id.g75687fc910_0_119
+---
+Round 2 was a continuation of the research started in [Accredited Bodies Research Round 1](link) where we hoped to gain further insights into the needs of Accredited Bodies around reporting.
+
+## Findings
+
+### Student Information Systems
+
+Accredited bodies have a student record information system that syncs automatically with UCAS.
+
+[Lookback highlight](https://docs.google.com/presentation/d/1ChxmFgCA3MHBxtcuDwtkiUeNdYBXs2prmeqa9jtQuk8/edit#slide=id.g6b4896b14d_1_124)
```

Comment:
Can these link directly to the clip on Lookback please

Reply:
I tracked down all but one of the clips in Lookback and have updated the missing Lookback link to read "Research Highlight" instead as it points to the playback.